packages/core/src/database/index.ts

```diff
@@ -1120,24 +1133,29 @@ $$ LANGUAGE plpgsql
 
           await sql
             .raw(`
-CREATE TRIGGER "${tableName}_reorg"
-AFTER INSERT OR UPDATE OR DELETE ON "${namespace}"."${tableName}"
-FOR EACH ROW EXECUTE FUNCTION ${tableName}_reorg_operation();
+CREATE TRIGGER "${sqlTableName}_reorg"
+AFTER INSERT OR UPDATE OR DELETE ON "${namespace}"."${sqlTableName}"
+FOR EACH ROW EXECUTE FUNCTION ${sqlTableName}_reorg_operation();
   `)
             .execute(qb.internal);
         }
       });
     },
     async revert({ checkpoint }) {
-      const tableNames = Object.values(args.schema)
-        .filter((table): table is PgTable => is(table, PgTable))
-        .map((table) => getTableConfig(table).name);
+      const sqlTableNames = getSQLTableNames();
+      const jsTableNames = getJsTableNames();
```

Comment:
I wonder if an API like `function getTableNames(): { js: string; sql :string }[]` would be safer